airbyte-commons-worker/src/test/java/io/airbyte/workers/normalization/DefaultNormalizationRunnerTest.java

```diff
@@ -49,6 +49,10 @@ class DefaultNormalizationRunnerTest {
   private static final String JOB_ID = "0";
   private static final int JOB_ATTEMPT = 0;
 
+  private static final String NORMALIZATION_IMAGE = "airbyte/normalization";
+  private static final String NORMALIZATION_TAG = "0.2.24";
```

Comment:
nitpick: let's use an obviously fake version number
```suggestion
  private static final String NORMALIZATION_TAG = "42.42.42";
```